src/app.md

```diff
@@ -234,6 +235,35 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-sizes]: #ex-file-sizes
+<a name="ex-file-sizes"></a>
+## Recursively calculate file sizes at given depth
+
+[min_depth](https://docs.rs/walkdir/1.0.7/walkdir/struct.WalkDir.html#method.min_depth) &
+[max_depth](https://docs.rs/walkdir/1.0.7/walkdir/struct.WalkDir.html#method.max_depth)
+usage example to calculate total file size at given depth.
```

Comment:
Please prefer prose explanation instead of comments in the source code. The prose description should be enough to understand what is going on in the code (just move or copy some of the comments to the description part).